### ISSUE TITLE
MacOs BlankPage after build

### ISSUE THREAD

**Issue by Cerceis** [nklayman/vue-cli-plugin-electron-builder#873]
**Describe the bug**
Everything run fine while on serve, but display blank page after building on macOs

**Screenshots**
Error displayed
<img width="239" alt="スクリーンショット 2020-06-30 18 35 37" src="https://user-images.githubusercontent.com/48720021/86110432-85f45d80-bb00-11ea-89e3-5b69d72ce9c5.png">

My vue Router setting:
<img width="292" alt="スクリーンショット 2020-06-30 18 37 38" src="https://user-images.githubusercontent.com/48720021/86110661-d075da00-bb00-11ea-8f48-02d05d9ce6a5.png">

background.js , basically unchanged only added ipc on top:
<img width="872" alt="スクリーンショット 2020-06-30 18 38 58" src="https://user-images.githubusercontent.com/48720021/86110872-0d41d100-bb01-11ea-885f-24455000a7a6.png">

**Environment:**
```
    "core-js": "^3.6.5",
    "fs": "0.0.1-security",
    "path": "^0.12.7",
    "vue": "^2.6.11",
    "vue-router": "^3.2.0",
    "vuex": "^3.4.0"
-dev:
    "@vue/cli-plugin-babel": "~4.4.0",
    "@vue/cli-plugin-router": "~4.4.0",
    "@vue/cli-plugin-vuex": "~4.4.0",
    "@vue/cli-service": "~4.4.0",
    "electron": "^9.0.0",
    "electron-devtools-installer": "^3.1.0",
    "vue-cli-plugin-electron-builder": "~2.0.0-rc.4",
    "vue-template-compiler": "^2.6.11"
```

**Comment by nklayman** [comment]
It works fine for me with the config you posted. Please create an example repo that demonstrates the error so that I can debug it.

**Comment by Cerceis** [comment]
> It works fine for me with the config you posted. Please create an example repo that demonstrates the error so that I can debug it.

I found the problem after further debugging.
I've set up my app to not load anything if there's no input data presented which is why it appears as blank page. The problem is localData is not getting compiled in build stage.

<img width="872" alt="スクリーンショット 2020-06-30 18 38 58" src="https://user-images.githubusercontent.com/48720021/86191993-6d795700-bb83-11ea-84f0-673f279a7df2.png">

I basically made a 'data' folder in root directory and read write from a JSON file inside.  Everything works fine in serve. but during build stage electron can't seem to compile or read the data.
